src/main/java/picard/util/help/PicardHelpDocWorkUnitHandler.java

```diff
@@ -50,10 +71,23 @@ protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
 
         // Picard tools use the summary line for the long overview section, so extract that
         // from Picard tools only, and put it in the freemarker map.
-        Class<?> toolClass = currentWorkUnit.getClazz();
-        if (picard.cmdline.CommandLineProgram.class.isAssignableFrom(toolClass)) {
+        final Class<?> clazz = currentWorkUnit.getClazz();
+        if (picard.cmdline.CommandLineProgram.class.isAssignableFrom(clazz)) {
             final CommandLineProgramProperties clpProperties = currentWorkUnit.getCommandLineProperties();
-            currentWorkUnit.setProperty("summary", clpProperties.summary());
+            currentWorkUnit.setProperty(WORK_UNIT_SUMMARY_KEY, clpProperties.summary());
+        } else if (MetricBase.class.isAssignableFrom(clazz)) {
+            currentWorkUnit.setProperty(WORK_UNIT_SUMMARY_KEY, currentWorkUnit.getSummary());
+            final List<Map<String, String>> metricsFields = new ArrayList<>();
+            currentWorkUnit.setProperty(METRICS_MAP_ENTRY_KEY, metricsFields);
+            final FieldDoc[] fieldDocs = currentWorkUnit.getClassDoc().fields(false);
+            for (final FieldDoc fd : fieldDocs) {
+                if (fd.isPublic()) {
+                    final Map<String, String> metricsField = new HashMap();
+                    metricsField.put(METRICS_MAP_NAME_KEY, fd.name());
+                    metricsField.put(METRICS_MAP_SUMMARY_KEY, fd.getRawCommentText());
+                    metricsFields.add(metricsField);
```

Comment:
Hm. Reviewing you own code days later is interesting. `metricsFields` -> `metricsList` and `metricsField` -> `metricsFields`.

Reply:
Fixed.